packages/mdc-chips/chip/foundation.js

```diff
@@ -109,6 +112,24 @@ class MDCChipFoundation extends MDCFoundation {
     this.shouldRemoveOnTrailingIconClick_ = shouldRemove;
   }
 
+  /**
+   * Computes and returns the bounding rect for the chip.
+   * @return {!ClientRect}
+   */
+  computeBoundingRect() {
+    // When a chip has a checkmark and not a leading icon, the bounding rect changes in size depending on the current
+    // size of the checkmark.
+    if (!this.adapter_.hasLeadingIcon() && this.adapter_.getCheckmarkBoundingClientRect()) {
```

Comment:
`this.adapter.getCheckmarkBoundingClientRect()` returns an object...I think it is more precise to check that the function call returns an object, instead of just calling the function.

Reply:
Changed it to check to confirm it isn't null, does that work?